src/Pages/Main/SectionRender/SectionRender.component.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import SpecialProductsSet from "../SetComponents/SpecialProductsSet/SpecialProductsSet.component";
+import EventsSet from "../SetComponents/EventsSet/EventsSet.component";
+import MDRecommend from "../SetComponents/MDRecommend/MDRecommend.component";
+import "./SectionRender.styles.scss";
+
+class SectionRender extends Component {
+  render() {
+    const { section } = this.props;
+    return (
+      <div
+        className={
+          section.componentType === "md"
+            ? "SectionRender md"
+            : section.componentType === "events" ||
+              section.componentType === "recipe"
+            ? "SectionRender other"
+            : "SectionRender products"
+        }
+      >
+        <Link
+          className={section.isLinkToAvailable ? "Link" : "Link null"}
+          to={section.isLinkToAvailable ? section.linkTo : ""}
+        >
+          <h2 className={"special-section-heading " + section.description}>
+            {section.title}
+            <div
+              className={
+                section.isLinkToAvailable
+                  ? "right-angle-icon"
+                  : "right-angle-icon hide"
+              }
+            ></div>
+          </h2>
+          <p
+            className={
+              section.subtitle.length > 0
```

Comment:
```suggestion
            className={
              !!section.subtitle.length
```
결국 true일 경우에 관한 것이므로 이렇게 표현해줄 수 있습니다

Reply:
오호 네 참고하곘습니다~